src/test/java/pageobjects/MainPage.java

```diff
@@ -0,0 +1,22 @@
+package pageobjects;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class MainPage {
+    private WebDriver driver;
+
+    @FindBy(xpath = "//*[contains(@title, 'Log in to your customer account')]")
```

Comment:
Selector by title doesn't seem a best one from my pov. Title is more human-readable thing, it can change in time.
Would it be better to have `.header_user_info a` or smth like that instead?